src/figuresClasses.ts

```diff
@@ -1,19 +1,71 @@
 export interface Figure {
-
+  shape: 'triangle' | 'circle' | 'rectangle',
+  color : 'red' | 'green' | 'blue',
+  getArea: () => number;
 }
 
 export class Triangle implements Figure {
+  shape = 'triangle';
```

Comment:
If you have TypeScript error while using this notation, you can add type:
```suggestion
  shape: 'triangle' | 'circle' | 'rectangle' = 'triangle';
```

And maybe storing it in another custom type would make it cleaner, at least that was my solution :)

Reply:
Yes, it's best to use another type here
```
export type Shape = 'triangle' | 'circle' | 'rectangle'
```
then in class use the type in the parameter
```
export class Triangle implements Figure {
  shape: Shape = 'triangle';
}
```

Of course you can do `Figure['shape']` as type annotation but it's ugly, it's better to export another type.

Reply:
And in `Figure` use 
```
export interface Figure {
  shape: Shape,
  color : 'red' | 'green' | 'blue', //color can be also exported as another type
  getArea: () => number;
}
```
Rule of thumb is to use as much types as you can in my opinion so don't hestitate to split it further